src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml.cs

```diff
@@ -90,12 +90,15 @@ private async Task Initialize()
             }
         }
 
-        private async void ZoomToFeatures(object sender, RoutedEventArgs e)
+        private async void StatesComboBox_SelectionChanged(object sender, RoutedEventArgs e)
         {
             try
             {
+                // Abbreviate the selected state alike the dataset's State field.
```

Comment:
`// Get the abbreviated letters of the currently selected state, these match the dataset's State field.`